packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -1,57 +1,75 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-meter-background-color: Specifies the background color of the component.
+ * @prop --calcite-meter-border-color: Specifies the border color of the component and displayed step lines.
+ * @prop --calcite-meter-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-meter-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-meter-fill-color: Specifies the color of the component's fill.
+ * @prop --calcite-meter-range-text-color: Specifies the color of the component's range labels.
```

Comment:
is there a request for the text in the various elements within meter to be different colors? Otherwise I would simplify this to `--calcite-meter-text-color`

Reply:
We can definitely remove `--calcite-meter-unit-text-color` and defer to the others.

This kind of override seems pretty desirable, I'd prefer to keep the `value-text` and `range-text`, I see them as pretty analogous to individual `heading-text` and `description-text` overrides.

<img width="478" alt="Screenshot 2024-02-09 at 10 37 37 AM" src="https://github.com/Esri/calcite-design-system/assets/4733155/8098d0be-6757-4f29-9091-375a77886250">

Reply:
Removed `--calcite-meter-unit-text-color` and simplified use of the other two.